sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -302,6 +302,35 @@ def chain_after(result):
 result.destination_copy_jobid_pairs <--> result['destination_copy_jobid_pairs']
 ```
 
+Writing with Storage Write API using Cross Language
+---------------------------------------------------
+After starting up an expansion service that contains the Java implementation
```

Comment:
Users should be able to just use the default expansion service, right ? Starting up a custom expansion service is a more advanced use-case which probably should not be mentioned here for clarity.

Reply:
Agreed